orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -1,386 +1,425 @@
 {
```

Comment:
<p>Beneat title, please write in normal font:</p><p><em>Prepared by: </em><a href="LinkedIn?" target="_blank">Your name</a> and [Angadh Nanjangud](<a href="https://www.angadhn.com/" target="_blank">https://www.angadhn.com/</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>